scripts/build-client-js.sh

```diff
@@ -11,6 +11,7 @@ cross-env BABEL_ENV="browser" browserify -t [babelify] \
 		index.js \
 		registrationDetails.js \
 		revision.js \
+		revisionsListPage.js \
```

Comment:
This file needs to be renamed: revisionsListPage.js -> revisions.js, and so needs changing in a few places in this file